### PR TITLE
allow for 3 github tokens

### PR DESCRIPTION
**Pull request by tyler17**
Default to first token if 2nd or 3rd token not provided

### CODE REVIEW

File: modules/delegates/api/fetchGithubDelegates.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ async function extractGithubInformation(
   folder: GithubPage
 ): Promise<DelegateRepoInformation | undefined> {
   try {
-    const folderContents = await fetchGitHubPage(owner, repo, folder.path);
+    const folderContents = await fetchGitHubPage(owner, repo, folder.path, 1);
```
Comment (b-pmcg) [inline modules/delegates/api/fetchGithubDelegates.ts:16]: Nice, think we could put the ID's into an enum? Might be easier to read if it's something like `GithubKeys.Delegates` or something similar.

Comment (dux-user) [reply modules/delegates/api/fetchGithubDelegates.ts:16]: Sounds good 👍

Comment (tyler17) [reply modules/delegates/api/fetchGithubDelegates.ts]: Good suggestion, I added the enum, and realized we should just use 3 different tokens, there's two different functions where we fetch the delegates folder, so just using the same token for both of those @b-pmcg

**Review by adamgoth — APPROVED** [review]
Good call, this should help reduce the chance of rate-limiting

**Review by rafinskipg — APPROVED** [review]

**Pull request merged**